test/js/node/child_process/child_process.test.ts

```diff
@@ -214,6 +215,19 @@ describe("spawn()", () => {
     const { stdout } = spawnSync("echo", ["hello"], { encoding: "utf8" });
     expect(stdout.trim()).toBe("hello");
   });
+
+  test.if(isLinux)("should handle null for first 4 positions of stdio", async () => {
+    // Run a program that does nothing except pause indefinitely, thus an
+    // opportunity to inspect its file descriptors.
+    const proc = spawn("sleep", ["inf"], { stdio: [null, null, null, null] });
+    // Delay here because the command (above) opens and closes a bunch of
+    // files when starting up, so we have to wait for it to stabilize.  This
+    // still is a race condition though, would be nice to eliminate.
+    await Bun.sleep(10);
```

Comment:
I love these comments @dhbaird! Can you expand the one " thus an opportunity to inspect its file descriptors." and explain what we are inspecting them for. To a casual reader getting back "0", "1", "2" may be opaque to the test of supporting 4 positions.